File: go-grpc/server/internal/dto/employeeDto.go
package dto

type EmployeeDto struct {
	Id           int              `json:"id"`
	Name         string           `json:"name"`
	EmployeeType string           `json:"employeeType"`
	Dob          *string          `json:"dob"`
	Certificates []CertificateDto `json:"certificates"`
}

File: flutter_app/lib/src/dto/employee_dto.dart
import 'package:flutter_app/src/dto/certificate_dto.dart';

class EmployeeDto {
  EmployeeDto(this.id, this.name, this.employeeType, this.dob);

  final int id;
  final String name;
  final String employeeType;
  final int? dob;
  List<CertificateDto> certificates = [];
}
